app/main.py

```diff
@@ -3,10 +3,26 @@
 from fastapi import FastAPI
 from stockfish import Stockfish
 from fastapi.responses import PlainTextResponse
+from pydantic import BaseModel, Field
+
+
+class EvaluationModel(BaseModel):
+    fen: str
+
+
+class AnalysisModel(BaseModel):
```

Comment:
`AnalysisRequest`?

Reply:
Also, since `AnalysisRequest` also seems to use the `fen` property, but adds the `move` property on top, it can just extend `EvaluationRequest` as such:

```suggestion
class EvaluationRequest(BaseModel):
    fen: str


class AnalysisRequest(EvaluationRequest):
    move: str
```